pipeline_noted.ipynb

```diff
@@ -14,45 +14,56 @@
+    "-  a [hyperlink](https://en.wikipedia.org/wiki/Hyperlink) usually point to a well-defined python module or class or methods, especially when that concept is first encountered in this document and the reader would likely need some background reference. The link usually poing to the official documentation of that concept, which might not be the best place to start for beginner. If you find the documentation puzzling, try to google that concept and find a tutorial that best suit your preference.\n",
     "-  an inline `code` usually refer to a name that already exsist in the [namespace](https://docs.python.org/3/tutorial/classes.html#python-scopes-and-namespaces) (i.e. the context where we run the codes in this document). It can be a previously encountered concepts, but more often it referes to variable or method names that we [imported](https://docs.python.org/3/reference/import.html) or defined along the way.\n",
-    "-  **bold** texts are used more loosely to highlight anything that requires more attention than plain texts. Though it's not used as carefully as previous formats, it often refer to some specific values that a variable or method arguments can assume."
+    "-  **bold** texts are used more loosely to highlight anything that requires more attention than plain texts, for example, the direct insructions for users. Though it's not used as carefully as previous formats, it often refer to some specific values that a variable or method arguments can assume."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Setting up"
+    "<font size=\"6\">**Workflow**</font>\n",
+    "\n",
+    "As shown in the workflow below, there are 7 main sections in this pipeline, and results data will be saved after **Motion correction**, **Background removal**, **Initialization**, and **CNMF**. When you run through this and decide to restart/shutdown the kernal before you finish the whole pipeline, you don't have to re-run everything (besides of **Setting Up** module) the next time you want to pick this up! For example, you restart/shutdown the kernal after **Initialization**, next time you can simply rerun the **Setting Up**, and go directly to **CNMF**.\n",
+    "\n",
+    "![workflow](Workflow.PNG)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The cells under this section should be executed every time the kernel is restarted."
+    "# <font size=\"9\">**Setting up**</font>\n"
```

Comment:
this doesn't seem to show up correctly at least on github. Can you double check on jupyter?